spec.html

```diff
@@ -37544,7 +37544,22 @@ <h1>Array.prototype.sort ( _comparefn_ )</h1>
               1. Append _kValue_ to _items_.
             1. Set _k_ to _k_ + 1.
           1. Let _itemCount_ be the number of elements in _items_.
-          1. [id="step-array-sort"] Sort _items_ using an implementation-defined sequence of calls to SortCompare. If any such call returns an abrupt completion, stop before performing any further calls to SortCompare or steps in this algorithm and return that completion.
+          1. [id="step-array-sort-sortcompare"] <a name="sec-sortcompare"></a> Let _sortCompare_ be a new Abstract Closure with parameters (_x_, _y_) that captures _comparefn_ and performs the following steps when called:
```

Comment:
`<a name="sec-sortcompare"></a>` is so that existing links to the spec with `#sec-sortcompare` continue working:
- Is that a goal we have?
- Is there a better way to do it?

Reply:
I've seen other sections with an `oldids` attribute, though not sure if that would work in this case.